### DISCUSSION TITLE
Support re-ordering signal columns

### DISCUSSION THREAD

**Discussion by CrossBound** [datalust/seq-tickets#1094] [Ideas]
Currently when editing a signal you can only add/remove a column, but you cannot re-order them. This means that if you need to change the arrangement you have to remove them all and re-add them. The problem is that when re-adding them you have to find an event with the property and add it from the event. 

It would be nice if you could simply re-arrange their order without adding/removing columns.

**Comment by nblumhardt** [comment]
Thanks for the suggestion, we'll give this one some thought 👍
